openedx/core/lib/token_utils.py

```diff
@@ -94,8 +98,17 @@ def attach_profile_claim(self, payload):
         except UserProfile.DoesNotExist:
             name = None
 
+        language = None
+        try:
+            user_preference = UserPreference.objects.get(user=self.user, key='pref-lang')
```

Comment:
`self.user.preferences.objects.get(key='pref-lang')` should also bring back the same thing but allows us to potentially optimize the query by joining the user preference table with user.